### PR TITLE
feat: [CDS-68601]:removed-encypted-text-supporting-harness-store

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Removed text supporting from encrypted file field for harness store, in config files.

#### Screenshots
Screenshot 2023-05-17 at 17 30 29

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger coverage

**Comment by litvinovstas** [comment]
retrigger titlecheck

File: src/modules/31-filestore/components/MultiTypeFileSelect/EncryptedSelect/EncryptedFileSelectField.tsx [hunk 1]
```diff
@@ -42,11 +42,11 @@ interface EncryptedData {
 
 function EncryptedFileSelectField(props: SelectEncryptedProps): React.ReactElement {
   const { getString } = useStrings()
-  const { formik, name, readonly = false, onChange, value = '', isSshWinRm = false } = props
+  const { formik, name, readonly = false, onChange, value = '' } = props
   const secretValue = get(formik.values, name) || ''
   const { openCreateOrSelectSecretModal } = useCreateOrSelectSecretModal(
     {
-      type: isSshWinRm ? 'SecretFile' : undefined,
+      type: 'SecretFile',
```
Comment (PratyushGar) [inline src/modules/31-filestore/components/MultiTypeFileSelect/EncryptedSelect/EncryptedFileSelectField.tsx:49]: Do we have an enum for this which can be used for 'SecretFile'?

**Review by PratyushGar — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**